src/q5/dispo-ELEC1330/summary/dispo-ELEC1330-summary.tex

```diff
@@ -579,4 +579,4 @@ \section{Question 11}
 \end{equation}
 La solution montre que le retour à l'équilibre après une faible perturbation s'effectue selon une fonction exponentielle décroissante du temps. La constante de temps de l'exponentielle est $\epsilon _s / \sigma$, ce qui est de l'ordre de $10^{-12}s$. Le retour à l'équilibre est donc très rapide. Ceci justifie, à postériori, les approximations faites. En effet, à ces échelles de temps, la recombinaison et la diffusion sont pratiquement inactives.
 
-\end{document}
+\end{document}
```

Comment:
You added a space here at the end of the line.

Reply:
Do you know how I can fix the problem? There is no space at this line. When I added a file in git, it told me that
"warning: LF will be replaced by CRLF in src/q5/dispo-ELEC1330/summary/dispo-ELEC1330-summary.tex.
The file will have its original line endings in your working directory."
That's why I wrote "git config --global core.autocrlf false" and the message doesn't appear anymore. But I still see the change at the last line on github...

Reply:
In my opinion : If you're on Windows, you shouldn't turn of autocrlf : this is because the original file, stored in Git, uses LF linefeed UNIX-style line ending, and not CRLF (carriage return - linefeed). But on Windows, text files use CRLF. To prevent conflicts, by default, Git on Windows adds CRLF to all files when checked out locally (to work with the normal line endings of the platform), and removes the CRLF when committing a text file with it, such that internally Git only deals with LF.
If you turn off this behavior, and you accidentally replace a LF by a CRLF when working with the file, it may get the CRLF back into Git, and it will thus appear as a modification.

Reply:
OK, but even if I let autocrlf, the problem is still there. :)

Reply:
I just checked it and actually, this is because you removed the line ending itself : there was a LF (in Git)/CRLF (checked ou on Windows), and it is now missing.

Reply:
Do you know how I can add this line ending?

Reply:
I think that's ok! Thanks